src/files.js

```diff
@@ -14,25 +14,27 @@ const { _COMMIT_CHANGELOG_CONTEXT_PATH: CONTEXT_PATH } = global;
  * @param {object} parsedCommits
  * @param {*|string} packageVersion
  * @param {object} options
- * @param {string} options.fallbackPackageVersion
- * @param {string} options.filePath
- * @param {string} options.headerMd
+ * @param {string} options.comparePath
+ * @param {string} options.date
  * @param {boolean} options.isDryRun
+ * @param {string} options.remoteUrl
+ * @param {object} settings
+ * @param {string} settings.fallbackPackageVersion
+ * @param {string} settings.filePath
+ * @param {string} settings.headerMd
+
  * @returns {string}
  */
 const updateChangelog = (
   parsedCommits = {},
   packageVersion,
+  { comparePath, date, isDryRun = false, remoteUrl } = OPTIONS,
   {
-    comparePath,
-    date,
     fallbackPackageVersion = '¯\\_(ツ)_/¯',
-    filePath = join(CONTEXT_PATH, `/CHANGELOG.md`),
+    filePath = join(OPTIONS.contextPath, `/CHANGELOG.md`),
     getComparisonCommitHashes: getAliasComparisonCommitHashes = getComparisonCommitHashes,
```

Comment:
required: confirm these funcs aren't passing options, may be able to cut down on additional drilling